serving/docs/lmi/user_guides/chat_input_output_schema.md

```diff
@@ -0,0 +1,259 @@
+# Chat Completions API Schema
+
+This document describes the API schema for the chat completions endpoints (`v1/chat/completions`) when using the built-in inference handlers in LMI containers.
+This schema is applicable to our latest release, v0.27.0.
```

Comment:
This line will go out of date quickly. Maybe refer to it as "v0.27.0 and later" so it will be correct until we deprecate the feature

Reply:
Changed.